app/src/main/java/app/revanced/integrations/patches/LithoFilterPatch.java

```diff
@@ -258,16 +258,11 @@ public GeneralBytecodeAdsPatch() {
     public boolean filter(final String path, final String identifier) {
         // Do not block on these
         if (ReVancedUtils.containsAny(path,
+                "comment_thread",
```

Comment:
What is this?

Reply:
The name used by comments component (when you open comment section and scroll comments).